packages/shared-ui/src/components/data-grid/tabulator-theme.css

```
@@ -0,0 +1,815 @@
+	border-radius: 20px;
+	font-size: 11.5px;
+	font-weight: 600;
+	border: 1px solid transparent;
+	white-space: nowrap;
+}
+.rr-cell-badge::before {
+	content: '';
+	width: 7px;
+	height: 7px;
+	border-radius: 50%;
+	background: currentColor; /* dot follows the variant text colour */
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)